app/models/miq_server.rb

```diff
@@ -544,4 +545,16 @@ def zone_unchanged_in_pods
 
     errors.add(:zone, N_('cannot be changed when running in containers')) if zone_id_changed?
   end
+
+  def messaging_health_check
+    broker = MiqQueue.messaging_client("health_check")
+    return if broker.nil?
+
+    _log.info("Initiating messaging health check")
```

Comment:
This is going to be logged every 60 seconds, I think we should only log on the failure path to not fill up the logs